js/src/components/ads-account-select-control/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import AppSelectControl from '.~/components/app-select-control';
+import useExistingGoogleAdsAccounts from '.~/hooks/useExistingGoogleAdsAccounts';
+
+/**
+ * @param {Object} props The component props
+ * @param {string} [props.value] The selected value. IF no value is defined, then the first option is selected and onChange function is triggered.
+ * @param {Function} [props.onChange] Callback when the select value changes.
+ * @return {JSX.Element} An enhanced AppSelectControl component.
+ */
+const AdsAccountSelectControl = ( {
+	value,
+	onChange = () => {},
```

Comment:
There seems no need to destructure these two props as they are just directly forwarded to `AppSelectControl` and whether the `onChange` needs a fallback will depend on the possible changes in (to be added).

Reply:
👍